mobile/src/main/java/org/openhab/habdroid/ui/WidgetAdapter.kt

```diff
@@ -354,6 +355,7 @@ class WidgetAdapter(
             Widget.Type.Mapview -> TYPE_LOCATION
             Widget.Type.Input -> if (widget.shouldUseDateTimePickerForInput()) TYPE_DATETIMEINPUT else TYPE_INPUT
             Widget.Type.Buttongrid -> TYPE_BUTTONGRID
+            Widget.Type.Button -> TYPE_BUTTON
```

Comment:
```suggestion
            Widget.Type.Button -> TYPE_INVISIBLE
```

Then you can remove the unused `TYPE_BUTTON`.

Reply:
I am right in the middle of tracking down a weird problem related to this. 

Basically the new buttons from Buttongrid are actual widgets. This means that the server will send state updates about the button (widget), so they need to be included in the main list of widgets. Therefore I believe this needs to stay here because it's used in places. I'm still trying to understand the big picture of the app, didin't realise I'd have to delve into it this deep.